### PR TITLE
feat(highlights configuration) - Add e2e tests for highlight configuration

### PR DESCRIPTION
**Pull request by ioanpocol**
Added e2e tests for highlights configurations
Changed the prepopulation script
Added unicity on highlight configuration name

### CODE REVIEW

File: client/app/scripts/superdesk-highlights/highlights.js [hunk 1]
```diff
@@ -206,9 +207,21 @@
...
                 $scope.modalActive = false;
-            }, function(result) {
-                $scope.message = gettext('There was a problem while saving configuration');
+            }, function(response) {
+                errorMessage(response);
             });
+            
+            function errorMessage(response) {
+                if (response.data && response.data._issues && response.data._issues.name && response.data._issues.name.unique) {
+                    $scope.message = gettext(
```
Comment (petrjasek) [inline client/app/scripts/superdesk-highlights/highlights.js]: this can't be translated, it's creating different string ids for each config name

**Comment by petrjasek** [comment]
tests look good :+1:

File: client/app/scripts/superdesk-highlights/highlights.js (hunk 1 above)
Comment (ioanpocol) [inline client/app/scripts/superdesk-highlights/highlights.js]: fixed

**Comment by petrjasek** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**